smtprelay.ini

```diff
@@ -1,7 +1,10 @@
 ; smtprelay configuration
 
 ; Logfile
-;logfile = /var/log/smtprelay.log
+;logfile = /dev/stdout
+
+; Minimum log level to write to Logfile
```

Comment:
It would be helpful to specify the possible options for this configuration setting. Also, is `debug` really the best choice for a default log level?

`Warn` would be the equivalent default in Python `logging` land. As a systemd user, I was considering suggesting that `smtprelay` default to log nothing unless an error happened. This is of course a very debatable topic.

Reply:
Again, these were very opinionated changes we made so they made sense in our context - 100% agree that it may not be the best choice for everyone

Reply:
I would suggest keeping the logfile as it was before (same as in config.go) and set log_level to info.